cupy/_math/misc.py

```diff
@@ -201,8 +201,17 @@ def clip(a, a_min=None, a_max=None, out=None):
 absolute = _core.absolute
 
 
-# TODO(beam2d): Implement it
-# fabs
+fabs = _core.create_ufunc(
+    'cupy_fabs',
+    ('b->b', 'B->B', 'h->h', 'H->H', 'i->i', 'I->I', 'l->l', 'L->L', 'q->q',
```

Comment:
The implemented dtypes are similar to `cupy.square`. Would you please let me know if more dtypes are to be added or if any types are to be removed?
Reference: [Array-protocol type strings](https://numpy.org/doc/stable/reference/arrays.dtypes.html).

Thank you! (:

Reply:
I suppose `['e->e', 'f->f', 'd->d']` is the choice.
```
>>> import numpy; numpy.fabs.types
['e->e', 'f->f', 'd->d', 'g->g', 'O->O']
```
and exclude `'g'` (`float128`) and `'O'` (`object`) which CuPy do not support.